chef/site-cookbooks/wca/templates/env.production.erb

```diff
@@ -37,3 +37,13 @@ STRIPE_CLIENT_ID=<%=
     "production" => @secrets['STRIPE_CLIENT_ID'],
   }[node.chef_environment]
 %>
+
+AWS_ACCESS_KEY_ID=<%= @secrets["AWS_ACCESS_KEY_ID"] %>
+AWS_SECRET_ACCESS_KEY=<%= @secrets["AWS_SECRET_ACCESS_KEY"] %>
+AWS_BUCKET=<%=
+  {
+    "development" => "",
+    "staging" => "staging.worldcubeassociation.org",
+    "production" => "www.worldcubeassociation.org",
```

Comment:
The bucket name for production doesn't contain the `www.`, should I add modify it?

Reply:
I just saw a private email where @jfly asked me to do this, so I went ahead and I'll be merging and deploying the PR now.